healthcheck/healthcheck_backend.go

```diff
@@ -0,0 +1,27 @@
+package healthcheck
+
+import (
+	_ "github.com/lib/pq"
+	"github.com/twitchscience/rs_ingester/loadclient"
+	"github.com/twitchscience/rs_ingester/metadata"
+	"github.com/twitchscience/scoop_protocol/scoop_protocol"
+)
+
+type HealthCheckBackend struct {
+	postgresScoopConnection loadclient.Loader
```

Comment:
That's what it is called in main (line 95) so I stuck with the naming convention.  I also kept postgres in the name because It creates a diverging naming convention when I use pg.  I can continue to change this if necessary though.